index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+  <head>Survey Form 
+  <link rel="stylesheet" href="css/style.css">
+  </head>
+  <body>
+  <div id="main">
+    <h1 id="title">FreeCodeCamp Survey Form</h1>
+    <p id="description">Thank you for taking the time to help us improve the platform</p>
+	<div id="outer">
+    <form id="survey-form">
```

Comment:
- [ ] Good job adding the inline comments. However, kindly give a "Why" on the reason you gave for asking for such change. A more detailed response on the reason to use `section` instead of `div` will give the student a more clear view of why the change was asked 👍